docs/githubteacher.md

```diff
@@ -0,0 +1,5 @@
+# GitHub Teacher
+
+- **Name:** GitHub Teacher
+- **Occupation:** Teacher
+- **Location:** San Francisco
```

Comment:
What if we were more specific here?  There are lots of San Franciscos in the :earth_africa: